template.json

```diff
@@ -9,10 +9,26 @@
       "@types/react-dom": "17.0.0",
       "@types/jest": "26.0.15",
       "typescript": "4.1.2",
-      "web-vitals": "1.0.1"
+      "web-vitals": "1.0.1",
+      "eslint": "7.25.0",
+      "prettier": "2.2.1",
+      "@nimbl3/eslint-config-nimbl3": "2.1.1",
+      "eslint-plugin-prettier": "3.4.0",
+      "eslint-config-prettier": "8.3.0",
+      "eslint-plugin-import": "2.22.1",
+      "eslint-plugin-jsx-a11y": "6.4.1",
+      "eslint-plugin-react": "7.23.2",
+      "eslint-plugin-react-hooks": "4.2.0",
+      "@typescript-eslint/eslint-plugin": "4.22.1",
+      "@typescript-eslint/parser": "4.22.1"
+    },
+    "scripts": {
+      "lint": "eslint ./src --ext .ts,.tsx",
+      "lint:fix": "eslint ./src --ext .ts,.tsx --fix",
+      "prettier": "prettier \"src/**/*.{ts,html}\" --write"
     },
     "eslintConfig": {
-      "extends": ["react-app", "react-app/jest"]
```

Comment:
@rafayet-monon Can you point me out why are we removing this extension? 🤔

Reply:
> @rafayet-monon Can we also update the template files to fix the error? so we don't have to fix them after we generate the app 💭

@carryall updated in [f7537dc](https://github.com/nimblehq/react-templates/pull/17/commits/f7537dce8a83fe4d55be1ab7559077b9b8aa211c).

> Can you point me out why are we removing this extension? 🤔

It's not really necessary to keep it. That `eslintConfig` doesn't affect the lining in any way whatsoever. The reason behind this is because we are using `jest: true` or `'jest/globals': true` in the `env` of Eslint.

Also, removed the `"react-app"` as well as it is created by the `create-react-app` itself.

Added, `eslint-plugin-jest` as well for the `tests` folder specifically in this PR as we've seen in `Toyota Wallet` project that it conflicts with other mocha and chai assertions from cypress.